lib/index.js

```diff
@@ -104,5 +140,4 @@ export default class Pageres {
 	}
 }
 
-objectAssign(Pageres.prototype, EventEmitter.prototype);
-objectAssign(Pageres.prototype, require('./util'));
+Object.assign(Pageres.prototype, util);
```

Comment:
There aren't good ways to statically type dynamic operations on classes with Flow.

In general, I wouldn't encourage patterns that aren't themselves added to ES (at least not yet). Flow doesn't compile and have runtime effects the way TypeScript does. This avoids ending up incompatible with the spec but also means things like mixins don't really work.

You can get around this by making it less static.

```js
const util = {
  utilMethod(): string {
 	return "hi";
  }
};

class Foo {
  utilMethod = util.utilMethod; // if the performance of the constructor matters you can do: utilMethod(...args) { util.utilMethod(...args); }

  constructor() {
    (this: typeof util); // in the future this should be `Foo implements util`
  }
}
```